basic python scripts/swapTwoNum.py

```diff
@@ -0,0 +1,14 @@
+# Python program to swap two variables
+
+
+
+a = input('Enter value of a: ')
+b = input('Enter value of b: ')
+
+
+temp = a
+a = b
+b = temp
+
+print('The value of a after swapping: {}'.format(a))
+print('The value of b after swapping: {}'.format(b))
```

Comment:
Using f string instead of format makes the print more understandable and short.

print(f"The value of a after swapping: {a}") is equivalent to your code.
It may not look advantageous in this case, but when sentences get long with more variables, it is certain to come in handy. Just some tips for better python programming :-)